### DISCUSSION TITLE
Expense categories views

### DISCUSSION THREAD

**Discussion by mahalel** [ananthakumaran/paisa#40] [Ideas]
Hey, thank you very much for this software. I have seen the HN thread and spent the last few days getting my finances in and I am really excited about this tool's potential.

One thing I would love to be able to do, is the ability to have a visual representation of the subcategories in a top Expense category, by month/year or even average.

For example, if I have created the following:

```
Expenses:Food:Groceries
Expenses:Food:Social
Expenses:Food:Takeaways
Expenses:Food:Alcohol
```

In the main Expense screen, by Month I can see I am spending let's say $1000 on food. But I am not able to see how much of this is one of the subcategories, unless i go into the editor view where I can see a numerical breakdown (which is not as satisfying as a visual graph).

Would this be something that you may consider implementing if not too difficult please?

**Comment by ananthakumaran** [comment]
As of today you can select a single category (2nd level) and focus on that. I am not sure if you have discovered this yet (you have to click the legend), as the UX is not great.

![image](https://github.com/ananthakumaran/paisa/assets/149238/4f664940-a203-45c4-b7e6-a7e11b38eb66)


Coming to your question, yes this is not great, and ideally, we should show the breakdown of that category. I want this as well. I have to think about it more, especially what color to use for the subcategories and how to show them, etc. Yes, I would figure out and implement a solution for this, no promise on the timeline though.

**Reply by mahalel** [reply]
Thank you, I do not expect promises but thank you for considering and putting it on the radar. Again, I really appreciate the tool and the work you put in!

**Comment by alessandrobertani** [comment]
Hi, any news on this feature request? I also think it would a really nice feature to have. Thanks!

**Reply by priyajit4u** [reply]
I am also waiting for this feature
